pontoon/contributors/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path, register_converter
 from django.urls.converters import StringConverter
+from django.views.decorators.cache import cache_page
```

Comment:
[Django docs](https://docs.djangoproject.com/en/5.0/topics/cache/#specifying-per-view-cache-in-the-urlconf) recommend to specify the per-view cache in the URLconf rather than next to the view functions themselves.